src/ResXManager.Model/Configuration.cs

```diff
@@ -140,5 +158,36 @@ protected Configuration(ITracer tracer)
         [DefaultValue(null)]
         [ForceGlobal]
         public string? TranslatorConfiguration { get; set; }
+
+        /// <summary>
+        /// Map deprecated property to current state (for backwards compatibility)
+        /// </summary>
+        private void MapOldPrefixValuesToNew()

```

Comment:
Personally, I've never been a huge fan of adding conversion methods like this to be maintained solely for the purpose of backwards compatibility. I would rather the code be written in a way that this kind of conversion wouldn't be necessary, but the new desired user experience didn't seem to fit with an enum value anymore, so I believe switching from an enum to the individual boolean values is a worthwhile change going forward.

Reply:
Just keep the enum and add a new flag value - the data model does not need to be a 1:1 representation of the UI